src/index.html

```diff
@@ -1,14 +1,62 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home"> 
+          <img src="./images/logo.png" alt="moyo-logo">
+        </a>
+      </div>
+  
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              githref="#Laptops & Computers"
```

Comment:
```suggestion
              href="#laptopscomputers"
```
не використовуємо пробіли в лінках, також краще використовувати lowercase